packages/design-system/src/components/Review/Review.test.tsx

```diff
@@ -78,4 +79,18 @@ describe('Review', function () {
 
     expect(wrapper.classList).toContain('my-class');
   });
+
+  it('calls onClick function when clicked', () => {
```

Comment:
This was the only unique test ReviewLink.test.tsx had, so I ported it over here.